examples/graphql-interceptor-configurations/graphql_interceptor_configurations.md

```diff
@@ -0,0 +1,25 @@
+# GraphQL service - Interceptor configurations
+
+A GraphQL interceptor can be configured to change the behavior of an interceptor. It can be configured via the `graphql:InterceptorConfig` annotation. The `graphql:InterceptorConfig` includes the field `global`, which accepts a boolean value. The `global` field defines the scope of the interceptor. If the `global` field is set to `true`, the interceptor will be applied to each field and subfield of the service. If the flag is set to `false`, the interceptor will be applied only to the fields of the type but not to the subfields of the type. By default, the `global` flag is set to true. Use `global: false` to apply interceptor functionality only to an entry point of the GraphQL service.
+
+>**Note:** The scope configuration is applied only to the GraphQL `service interceptors`.
```

Comment:
Why are we removing this?

Reply:
Because this sentence is merged with the info section in the above review suggestion. We don't need to have 2 info and note sections here if we can merge both as shown above.

Reply:
+1, added a small suggestion.